docs/source/online/fromBuilder.rst

```diff
@@ -1,170 +1,73 @@
-
-Keep checking the :ref:`onlineStatus` to see what is supported. You might want to sign up to the `PsychoPy forum <http://discourse.psychopy.org>`_ and turn on "watching" for the `Online Studies <http://discourse.psychopy.org/c/online>`_ category to get updates. That way you'll know if we update/improve something and you'll see if other people are having issues.
-
-.. _onlineExpSettings:
-
-Check your experiment settings
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-In your Experiment Settings there is an "Online" tab to control the settings.
-
-Path: When you upload your study to Pavlovia it will expect to find an 'html' folder in the root of the repository, so you want to set this up with that in mind. By default the output path will be for a folder called html next to the experiment file. So if that is in the root of the folder you sync online then you'll be good to go! Usually you would have a folder structure something like this and :ref:`sync that entire folder with pavlovia.org <pavloviaSync>`:
-
-.. figure:: /images/foldersStimHTML.png
-  :alt: Folder structure with the experiment (`blockedTrials.psyexp`), a `stims` folder in which the stimuli are stored, some conditions files and an `html` folder containing the code for the study to run online.
-
-.. _onlineExportHTML:
-
 Export the HTML files
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Once you've checked your settings you can either go to `>File>Export HTML` from the Builder view with your experiment open OR press 'sync' from the globe icons.
-
-Both of these will generate all the necessary files (HTML and JS) that you need for your study, however sync will also create a project in pavlovia.org
-
-
-.. _onlineUploadServer:
-
-Uploading files to your own server
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-We really don't recommend this and can only provide limited help if you go this route. If you do want to use your own server:
-
-You will need some way to save the data. PsychoJS can output to either:
-
-- `csv` files in `../data` (i.e. a folder called `data` next to the html folder). You'll need this to have permissions so that the web server can write to it
-- a relational database
-- You should make sure your server is using https to encrypt the data you collect from your participants, in keeping with GDPR legislation
-- You will need to install the server-side script
-- You will need to adapt PsychoPy Builder's output scripts (`index.html` and the `<experimentName>.js`) so that the references to `lib/` and `lib/vendors` are pointing to valid library locations (which you will either need to create, or point to original online sources)
-
-.. _onlineDebugging:
-
-Debug your online experiments
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To generate an online experiment from PsychoPy Builder you can either:
 
-This is going to be trickier (for now) than the PsychoPy/Python scripts. The main resources we recommend are:
+- go to `>File>Export HTML`, or
```

Comment:
I think this should definitely mention that users should use Runner and:

- look out for Alerts where we can warn about things e.g. that aren't going to work online (we haven't added enough of these yet but we should)
- use the debug-online-local option of Runner so they can debug their web experiment in a browser without constantly pushing to pavlovia